.github/workflows/nightly_run.yml

```diff
@@ -323,16 +323,13 @@ jobs:
           - make_options_run_mode: $FAST_JIT_BUILD_OPTIONS
             make_options_feature: "-DWAMR_BUILD_MEMORY64=1"
     steps:
-      - name: checkout
-        uses: actions/checkout@v3
```

Comment:
can't you just bump to actions/checkout@v4?

Reply:
The v4 will face the same issue, node 20 will require a more recent version of glibc and glibcxx than the Ubuntu 14.04 docker image provides

Reply:
ah, i totally misread the PR description. sorry for noise!

Reply:
No worries, I also made some modifications to my previous comments to make it clearer